pkg/kv/kvnemesis/kvnemesis.go

```diff
@@ -171,16 +174,28 @@ func RunNemesis(
 
 		reproFile := l(ctx, "repro.go", "// Reproduction steps:\n%s", printRepro(stepsByWorker))
 		rangefeedFile := l(ctx, "kvs-rangefeed.txt", "kvs (recorded from rangefeed):\n%s", kvs.DebugPrint("  "))
-		var kvsFile string
-		scanKVs, err := dbs[0].Scan(ctx, dataSpan.Key, dataSpan.EndKey, -1)
-		if err != nil {
-			l(ctx, "", "could not scan actual latest values: %+v", err)
-		} else {
+		kvsFile := "<error>"
+		var scanKVs []kv.KeyValue
+		for i := 0; ; i++ {
+			var err error
+			scanKVs, err = dbs[0].Scan(ctx, dataSpan.Key, dataSpan.EndKey, -1)
+			if errors.Is(err, errInjected) && i < 100 {
```

Comment:
What's the significance of 100? Just randomly picked to terminate an infinite loop?